docs/DeveloperGuide.md

```diff
@@ -404,6 +404,25 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 --------------------------------------------------------------------------------------------------------------------
 
+## **Appendix: Planned Enhancements**
+
+Team size: 5
+
+1. Currently, if any component (name, email, etc.) is too long, it will be hidden on the screen. We plan to implement
+   appropriate mechanism to show the full component.
+2. Currently, names like "`John Doe`" and "`John  Doe`" (two spaces in the middle) are considered different. We plan
+   to have more sophisticated duplication detection mechanism.
+3. Currently, `undo` only supports undoing one command. We plan to support undoing multiple commands.
+4. Currently, a professor must have an address attached. This is reasonable in most cases; however, some visiting
+   instructor may not have a specific office address in NUS. Therefore, we plan to allow omitting the field if the user desires.
+5. Currently, there's no way to make `search` more targeted; for example, if there are 3 entries in the address book with name `John Doe`, `Jane Doe` and `John Smith`, then there's no way to search for specifically `John Doe`. In the future, we plan to have a way to specify that multiple words must match exactly, and have a priority system where contacts that matches more keywords are shown earlier.
+6. Currently, some special characters such as `/` cannot appear in entries like name or address. We plan to have
```

Comment:
I think we can do / for name and address without do f/, see image below. But we can't do it for tag.
![image](https://github.com/AY2324S2-CS2103T-T11-2/tp/assets/122252489/6d2ae8b1-d09f-415b-8ed9-f1a73623f71e)

Reply:
No, the thing is **sometimes** it will fail to be added. But let me clarify.